lib/data.dart

```diff
@@ -0,0 +1,92 @@
+import 'dart:convert';
+import 'dart:io';
+
+Future<List<ObjectInfo>> requestImages() async {
+  try {
+    Uri apiUri = Uri.parse(
+        'https://api.harvardartmuseums.org/object?sort=random&apikey=b1e1f600-aa67-11e9-adb9-2fdc07e84b9c');
+
+    HttpClientRequest request = await HttpClient().getUrl(apiUri);
+    HttpClientResponse response = await request.close();
+
+    Stream resStream = response.transform(utf8.decoder);
+    var result = '';
+    await for (var data in resStream) {
+      result += data as String;
+    }
+
+    List<ObjectInfo> output =
+    MuseumResponse.fromJson(json.decode(result) as Map<String, dynamic>)
```

Comment:
```suggestion
    .fromJson(json.decode(result) as Map<String, dynamic>)
```

Reply:
I agree it's better to read but the travis script will only pass if I use the default auto formatting.

Reply:
Cool! I'm still a bit unfamiliar with the Travis rules. As you know, there's a very different style guide for flutter/flutter :)